asv_bench/benchmarks/utils/common.py

```diff
@@ -456,7 +456,8 @@ def trigger_import(*dfs):
     *dfs : iterable
         DataFrames to trigger import.
     """
-    assert ASV_USE_STORAGE_FORMAT == "omnisci"
+    if ASV_USE_STORAGE_FORMAT != "omnisci":
```

Comment:
to allow usage of `trigger_import` at the file with base benchmarks (`asv_bench/benchmarks/benchmarks.py`)

Reply:
ok